frontend/src/components/campAdmin/CampActivityProgressLabels.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <content-group>
+    <slot name="title">
+      <div class="ec-content-group__title py-1 subtitle-1">
+        {{ $tc('components.campAdmin.campActivityProgressLabels.title') }}
+        <dialog-activity-progress-label-create v-if="!disabled" :camp="camp()">
+          <template #activator="{ on }">
+            <button-add
+              color="secondary"
+              text
+              :hide-label="$vuetify.breakpoint.xsOnly"
+              class="my-n2"
+              v-on="on"
+            >
+              {{ $tc('components.campAdmin.campActivityProgressLabels.create') }}
+            </button-add>
+          </template>
+        </dialog-activity-progress-label-create>
+      </div>
+    </slot>
+    <v-skeleton-loader v-if="loading" type="article" />
+    <v-list>
+      <v-list-item
+        v-for="(progressLabel, idx) in progressLabels"
+        :key="progressLabel._meta.self"
+        class="px-0"
+      >
+        <v-list-item-content class="pt-0 pb-2">
+          <v-list-item-title>
+            {{ idx + 1 }}) {{ progressLabel.title }}
+          </v-list-item-title>
+        </v-list-item-content>
+
+        <v-list-item-action v-if="!disabled" style="display: inline">
+          <v-item-group>
+            <dialog-activity-progress-label-edit :progress-label="progressLabel">
+              <template #activator="{ on }">
+                <button-edit class="mr-1" v-on="on" />
+              </template>
+            </dialog-activity-progress-label-edit>
+          </v-item-group>
+        </v-list-item-action>
+
+        <v-menu v-if="!disabled" offset-y>
+          <template #activator="{ on, attrs }">
+            <v-btn icon v-bind="attrs" v-on="on">
+              <v-icon>mdi-dots-vertical</v-icon>
+            </v-btn>
+          </template>
+          <v-list>
+            <v-list-item @click="moveUp(progressLabel)">
+              <v-list-item-icon>
+                <v-icon>mdi-arrow-up-bold</v-icon>
+              </v-list-item-icon>
+              <v-list-item-title>UP</v-list-item-title>
+            </v-list-item>
+            <v-list-item @click="moveDown(progressLabel)">
+              <v-list-item-icon>
+                <v-icon>mdi-arrow-down-bold</v-icon>
+              </v-list-item-icon>
+              <v-list-item-title>DOWN</v-list-item-title>
+            </v-list-item>
+
+            <v-divider />
+
+            <dialog-entity-delete
+              :entity="progressLabel"
+              :error-handler="deleteErrorHandler"
```

Comment:
this did not work on the pr deployment. the label was not deleted, but i did not see a message

Reply:
Same for me